src/main/java/com/ca/apim/gateway/cagatewayconfig/tasks/zip/builder/JdbcConnectionEntityBuilder.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2018 CA. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+package com.ca.apim.gateway.cagatewayconfig.tasks.zip.builder;
+
+import com.ca.apim.gateway.cagatewayconfig.tasks.zip.beans.Bundle;
+import com.ca.apim.gateway.cagatewayconfig.tasks.zip.beans.JdbcConnection;
+import com.ca.apim.gateway.cagatewayconfig.util.IdGenerator;
+import com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentTools;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class JdbcConnectionEntityBuilder implements EntityBuilder {
+    private final Document document;
+    private final IdGenerator idGenerator;
+
+    public JdbcConnectionEntityBuilder( Document document, IdGenerator idGenerator) {
+        this.document = document;
+        this.idGenerator = idGenerator;
+    }
+
+    public List<Entity> build(Bundle bundle) {
+        return bundle.getJdbcconnections().entrySet().stream().map(jdbcConnectionEntry ->
+                buildEntity(bundle, jdbcConnectionEntry.getKey(), jdbcConnectionEntry.getValue())
+        ).collect(Collectors.toList());
+    }
+
+    private Entity buildEntity(Bundle bundle, String name, JdbcConnection jdbc) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused method parameter "bundle". ([squid:S1172](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS1172))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=53&issues=AWU6Y48aWWqqPQVjPd1z&open=AWU6Y48aWWqqPQVjPd1z)

[ISSUE_KEY]: AWU6Y48aWWqqPQVjPd1z